### PR TITLE
feat(examples)

### PR DESCRIPTION
**Pull request by SimonHofmeister**
add vanilla and vanilla-cdn example. Replace names of examples to fix errors

### CODE REVIEW

File: examples/vanilla-cdn/README.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# vanilla cdn
+
+## Project setup
+
+```
+npm install
+```
+
+### Compiles and hot-reloads for development
```
Comment (acstll) [inline examples/vanilla-cdn/README.md:9]: I would rephrase this to "Starts a local server with hot-reloads for development", since there's no compiling happening.

**Review by acstll — APPROVED** [review]
Thank you very much!

File: examples/vanilla-cdn/README.md (hunk 1 above)
Comment (SimonHofmeister) [reply examples/vanilla-cdn/README.md]: done
